src/background/Wallet/Wallet.ts

```diff
@@ -414,6 +417,23 @@ export class Wallet {
     return walletContainer.getFirstWallet();
   }
 
+  async uiAddReadonlyAddress(
+    params: WalletMethodParams<{ address: string; name: string | null }>
+  ) {
+    await this.uiImportReadonlyAddress(params);
```

Comment:
Why do we need this?

Reply:
You mean why do we need to add wallet to the "watched wallets" list? This is a product requirement

Reply:
Ah, I see. Let's add a comment in the service's code about this.

Reply:
I added a comment about this on the caller site